app/src/main/res/layout/activity_compose.xml

```diff
@@ -22,6 +22,19 @@
             tools:ignore="ContentDescription" />
         <!--content description will be set in code -->
 
+        <Spinner
+            android:id="@+id/composePostLanguageButton"
+            style="@style/TuskyImageButton"
+            android:layout_width="44dp"
```

Comment:
I think it needs to be set to something larger or even `wrap_content`

<img src="https://user-images.githubusercontent.com/10157047/187499299-c224fc28-1a75-4220-929b-f4b7bfcdf13e.png" width="360"/>

Reply:
Argh, what language is that?
The issue with `wrap_content` is that it wants to size itself for the dropdown items instead 😐.
Although if you know of a workaround (or if I'm just Doing It Wrong)…

Reply:
Ok, checked this variation against all languages installed in my emulator